src/Wallabag/ImportBundle/Import/PocketImport.php

```diff
@@ -194,10 +194,10 @@ public function parseEntry(array $importedEntry)
         $this->fetchContent($entry, $url);
 
         // 0, 1, 2 - 1 if the item is archived - 2 if the item should be deleted
-        $entry->setArchived(1 === $importedEntry['status'] || $this->markAsRead);
+        $entry->setArchived(1 == $importedEntry['status'] || $this->markAsRead);
```

Comment:
Maybe the best fix is to cast the value as integer:
```suggestion
        $entry->setArchived(1 === (int) $importedEntry['status'] || $this->markAsRead);
```